intermediate_source/torchvision_tutorial.rst

```diff
@@ -304,6 +304,30 @@ be using Mask R-CNN:
 That’s it, this will make ``model`` be ready to be trained and evaluated
 on your custom dataset.
 
+Checking the model with random tensors (Optional)
+---------------------------
+
+Before iterating over the dataset, it's always good to see what the model 
+expects during training and inference time with random tensors. 
```

Comment:
This is something we should be careful about: if we have batch normalization layers in the model (which is not the case for detection models because we freeze the batch norm layers, but happens in classification), feeding randomly-initialized data will affect the batch norm statistics, leading to potentially unexpected results

Reply:
I've now changed the code from using random tensors to sampled data from the actual dataset. This will fix the batch norm stats issue.